rudaux/course_api.py

```diff
@@ -92,6 +92,7 @@ def _canvas_get_people_by_type(config, course_id, typ):
     return [ { 'id' : str(p['user']['id']),
                'name' : p['user']['name'],
                'sortable_name' : p['user']['sortable_name'],
+               'course_section_id' : str(p['course_section_id']),
```

Comment:
does this always exist? I imagine for instructors it might not? You should probably check existence and fill with `None` otherwise

Reply:
also try probing the rest API directly to find out. Easy way to do that is to put the URL that I send via `requests` to get lists of people directly into the URL bar in your browser instead. You'll see raw JSON response in the browser, just look through that to check.

Reply:
I checked the API docs and it appears to be a required field. Also checked list of enrollments, looked like it was there for everyone.